### ISSUE TITLE
Inserting a nil into a text column results in 'false' (PostgreSQL)

### ISSUE THREAD

**Issue by andrei-dubovik** [fukamachi/cl-dbi#38]
E.g.,
(dbi:do-sql *dbh* "CREATE TABLE example (name text)")
(dbi:execute (dbi:prepare *dbh* "INSERT INTO example (name) VALUES (?)") nil)
yields a text string 'false' in the name column. So, is it intended behaviour?

**Comment by andrei-dubovik** [comment]
Yes, I suppose it is intended behaviour. I've just found out it's coming from cl-postgres:to-sql-string (see https://github.com/marijnh/Postmodern/blob/master/cl-postgres/sql-string.lisp). Keyword :null does the trick. Is it the same behaviour in mysql and other drivers?

**Comment by fukamachi** [comment]
Thank you for reporting.
Though CL-DBI should abstract the conversion data between DB and CL, it still remains to do in the future.
